pyproject.toml

```diff
@@ -34,14 +34,14 @@ documentation = "https://django-pghistory.readthedocs.io"
 [tool.poetry.dependencies]
 python = ">=3.6,<4"
 django = ">=2"
-django-pgconnection = "^1.0.0"
-django-pgtrigger = "^1.0.0"
+django-pgconnection = ">=1.0.0"
+django-pgtrigger = ">=2.1.0"
 
 [tool.poetry.dev-dependencies]
 black = "=19.10b0"
 boto3 = "=1.9.243"
 dj-database-url = "^0.5.0"
-django-dynamic-fixture = "^3.1.0"
+django-dynamic-fixture-dj31 = "=3.1.1"
```

Comment:
There's been an oustanding pr for ddf to have 3.1 compatibility for some time now. I went ahead and deployed a fork as a temporary fix and will revert this back once the author has released the fix